exes/ethercat/inc/public/tfc/ec.hpp

```diff
@@ -233,14 +232,6 @@ class context_t {
     min_cycle_ = std::min(min_cycle_, last_cycle_);
     max_cycle_ = std::max(max_cycle_, last_cycle_);
 
-    if (cycle_count_ % 10'000 == 0 and false) {
-      // log the max cycle time
-      logger_.trace("Ethercat max cycle time: {} us", max_cycle_.count());
-      logger_.trace("Ethercat max cycle time with sleep: {} us", max_cycle_with_sleep_.count());
-      logger_.trace("Ethercat min cycle time: {} us", min_cycle_.count());
-      logger_.trace("Ethercat min cycle time with sleep: {} us", min_cycle_with_sleep_.count());
-    }
-
```

Comment:
I would like to keep this for debugging purposes

Reply:
dbus signals?

or just make it log, remove the `and false`

Reply:
I was thinking ipc signals, but then again we are going to need some comprehensive way to debug the ethercat network. This could become one part of that and be exposed over dbus somehow.

Reply:
why not use logging for debugging purposes?